docs/content/en/docs/reference/clusterspec/optional/registrymirror.md

```diff
@@ -43,6 +47,18 @@ spec:
 * __Type__: string
 * __Example__: ```port: 443```
 
+### __ociNamespaces__ (optional)
+* __Description__: mapping from upstream registries to the locations of the private registry. When specified, the artifacts pulled from an upstream registry will be put in its corresponding location/namespace in the private registry. The target location/namespace must be already existing.
+* __Type__: array
+* __Example__: <br/>
+  ```yaml
+  ociNamespaces:
+    - registry: "public.ecr.aws"
+      namespace: "eks-anywhere"
+    - registry: "875320726801.dkr.ecr.us-west-2.amazonaws.com"
```

Comment:
```suggestion
    - registry: "783794618700.dkr.ecr.*.amazonaws.com"
```

Reply:
Both work, so I don't think customers need to do extra step.